AlphaWallet/Tokens/ViewModels/EthCurrencyHelper.swift

```diff
@@ -51,7 +51,7 @@ class EthCurrencyHelper {
     public func valueChanged24h(currencyAmountWithoutSymbol: Double?) -> String? {
         if let percentChange = percentageChange24h, let value = currencyAmountWithoutSymbol {
             let change = value * percentChange / 100
-            if let string = CurrencyFormatter.usdFormatter.string(from: NSNumber(value: change)) {
+            if let string = NumberFormatter.usd.string(from: change) {
```

Comment:
Sorry missed this one. The if-let-else isn't necessary. Just return `NumberFormatter.usd.string(from: change)`.